contrib/build-rpm.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
In case you've not seen it, I recommend https://google.github.io/styleguide/shell.xml